src/wpool_process_callbacks.erl

```diff
@@ -0,0 +1,76 @@
+-module(wpool_process_callbacks).
+
+-behaviour(gen_event).
+
+%% gen_event callbacks
+
+-export([ init/1
+        , handle_event/2
+        , handle_call/2
+        , handle_info/2
+        , code_change/3
+        , terminate/2]).
+
+-export([notify/3]).
+-type state() :: ordset:ordset(module()).
+
+-type event() :: on_init_start | on_new_worker | on_worker_dead.
+
+-spec init([wpool:option()]) -> {ok, state()}.
+init(WPoolOpts) ->
+{ok, maybe_initial_callbacks(WPoolOpts)}.
```

Comment:
indentation :)

Reply:
Also, if instead of managing a list of callbacks here, we let `gen_event` take care of that, we won't need to initialize the list of callbacks here. We should do it in `wpool_pool:start_link(…)`, probably after initializing the pool itself.